x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -258,10 +256,9 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
     cy.url().should('include', expectedVal);
   });
 
-  it(`Filters are restored from localstorage when user navigates back to the page.`, () => {
+  it(`should restore Filters from localstorage when user navigates back to the page.`, () => {
     cy.get(OPTION_LIST_VALUES(1)).click();
-    cy.get(OPTION_SELECTABLE(1, 'high')).should('be.visible');
-    cy.get(OPTION_SELECTABLE(1, 'high')).click({});
+    cy.get(OPTION_SELECTABLE(1, 'high')).click();
```

Comment:
line 265, do we have to `waitForPageFilters` here? The next actions are navigating away then back...

Reply:
yes, `waitForPageFilters` ensures that the value selected by user is synced to the url and local storage. Removing it has the potential to make test flaky.

Reply:
ok that makes sense, by the way, there is a typo in the word `scuccessfully` below :)

Reply:
Sorry for my spellings 😭